src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -99,7 +94,7 @@ private AIState startWorkingAtOwnBuilding()
 
     private AIState prepareForMining()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "prepareForMining" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)